mobile/mobile.go

```diff
@@ -84,51 +88,54 @@ var (
 
 	jsonCBRegex = regexp.MustCompile(`(\w+)\.(\w+)\.(\w+)`)
 
-	m *mobileClient
+	m = make(map[string]*mobileClient)
 
 	registry = make(map[string]func(context.Context,
 		*grpc.ClientConn, string, func(string, error)))
 
-	localPrivCreateCallback  NativeCallback
-	remoteKeyReceiveCallback NativeCallback
-	authDataCallback         NativeCallback
+	interceptorLogsInitialize = false
```

Comment:
do we not need to worry about concurrent access of these glabal variables?

Reply:
only thing we'd potentially worry about is `interceptorLogsInitialize` but that's kept in check by the locks on `signal.Intercept` from LND. the `perms` and `registry` should always be populated with the same data given the same node and version of LNC.

Reply:
I mainly meant the `m` map

Reply:
Hadn't even considered that'd be an issue. I will mess around with giving it a `RWMutex` and perhaps even using `syncmap` instead